ddl/ddl_api.go

```diff
@@ -122,7 +122,7 @@ func getDefaultCharsetAndCollate() (string, string) {
 	// TODO: TableDefaultCharset-->DatabaseDefaultCharset-->SystemDefaultCharset.
```

Comment:
I think we need to finish this TODO to solve the issue in the pr description.
We could return the values of "character_set_server", or "character_set_database" here.
Then we could run the SQL `set global/session character_set_server = 'utf8mb4'` to change the default character set.

Reply:
Right, this TODO issue you bring up could have provided me with alternative ways to try to fix this issue. Alter the database default charset probably would have worked, but would require altering my workflow.

I just reviewed the variable interaction of this PR more and pushed a commit to revert the change to `character_set_system`.

I agree in general with the philosophy that configuration should be update-able at runtime. However, in this case
1) there is no variable matching the default charset in MySQL, it is just a [startup or compile-time configuration](https://dev.mysql.com/doc/refman/8.0/en/charset-server.html)
2) it creates more code to maintain (with potential for race conditions)
3) I don't think there is a valid use case for changing the default charset at runtime. The only use case would seem to be recognizing a mis-configuration issue that you want to correct by changing once, which can be done via restart. For performance tuning variables, for example, you don't know what the correct configuration is and you have to tinker.

Let me know if that makes sense.

Reply:
OK.